cpu/cc2538/ldscripts/cc2538nf11.ld

```diff
@@ -25,6 +25,15 @@ MEMORY
     ram   (rwx) : ORIGIN = 0x20004000, LENGTH = 16K
 }
 
-INCLUDE cc2538.ld
+/* MCU Specific Section Definitions */
+SECTIONS
+{
+    .flashcca :
+    {
+        KEEP(*(.flashcca))
+    } > cca
+}
+
+INCLUDE cortexm_base.ld
```

Comment:
Looking back at it I think I included `cortexm_base.ld` at the end of the file for consistency with the other `cpu/*/ldscripts`. No technical reason for doing so.